### PR TITLE
#260 boots changed the name for HW client

### PR DESCRIPTION
**Pull request by Cbkhare**
## Description

Changed name of tinkerbell HW client 

## Why is this needed

Hw client in tink repo hsa been changed

Fixes: #

## How Has This Been Tested?
Has been tested along with PR https://github.com/tinkerbell/tink/pull/210 and by bringing worker up.

### CODE REVIEW

File: packet/client.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func NewClient(consumerToken, authToken string, baseURL *url.URL) (*Client, erro
 	dataModelVersion := os.Getenv("DATA_MODEL_VERSION")
 	switch dataModelVersion {
 	case "1":
-		hg, err = tinkClient.NewTinkerbellClient()
+		hg, err = tinkClient.TinkerbellHWClient()
```
Comment (kdeng3849) [inline packet/client.go:51]: I think it's better to write out the word `Hardware`. It's easier to read and more consistent with `TinkerbellWorkflowClient` in my opinion.

Comment (grahamc) [reply packet/client.go:51]: Agreed, good point Kelly!

Comment (Cbkhare) [reply packet/client.go]: done

**Comment by Cbkhare** [comment]
declined, will take forward change for this with https://github.com/tinkerbell/boots/pull/50

**Pull request closed without merging**